cylc/flow/network/schema.py

```diff
@@ -1712,12 +1719,17 @@ class Arguments:
 class Stop(Mutation):
     class Meta:
         description = sstrip('''
-            Tell a scheduler to shut down,
-            or stop a specified flow from spawning any further.
+            Tell a workflow to shut down.
 
             By default stopping workflows wait for submitted and running tasks
             to complete before shutting down. You can change this behaviour
             with the "mode" option.
+
+            Tasks that become ready after the shutdown is ordered will be
+            submitted immediately if the workflow is restarted.
+            Remaining task event handlers, job poll and kill commands, will
+            be executed prior to shutdown, unless
+            ``mode==WorkflowStopMode.Now.name``.
```

Comment:
```suggestion
            the stop mode is `{WorkflowStopMode.Now.name}`.
```
(Will need to covert the string into a f-string)